### ISSUE TITLE
Fill Model Analysis Page

### ISSUE THREAD

**Issue by Ivaldivi** [Ivaldivi/DATA515-Image-Classification#31]
To be completed **after** testing Tensorflow model
* Loss & val_loss
* Accuracy
* ROC (annotated AUROC)
* Precision recall curve
* Train test split description
* Transfer learning (what model did we use and why?) 
* 4 personal photo examples
* Accuracy Cutoff Calculation

**Comment by anthony-nguyen-04** [comment]
https://github.com/Ivaldivi/DATA515-Image-Classification/pull/53
